lib/model/model.go

```diff
@@ -1914,7 +1909,7 @@ func (m *Model) internalScanFolderSubdirs(ctx context.Context, folder string, su
 		// The error we get here is likely an OS level error, which might not be
 		// as readable as our health check errors. Check if we can get a health
 		// check error first, and use that if it's available.
-		if ferr := m.CheckFolderHealth(folder); ferr != nil {
+		if ferr := runner.CheckHealth(); ferr != nil {
 			err = ferr
 		}
 		runner.setError(err)
```

Comment:
Isn't this pointless?

Reply:
Not sure, the reason given in the comments seems sound enough to me?

Reply:
A right, if it's err not ferr, it wouldn't be set otherwise.